Framework/Kernel/inc/MantidKernel/ITimeSeriesProperty.h

```diff
@@ -45,6 +45,9 @@ class ITimeSeriesProperty {
   // Using property seemed to be the most straightforward solution.
   virtual Property *cloneWithTimeShift(const double timeShift) const = 0;
 
+  /// Create a partial copy according to TimeROI
```

Comment:
could you write in some detail what "according to TimeROI" means?

Reply:
Done